code/app/src/main/java/com/cmput301w21t06/crowdfly/Controllers/ExperimentLog.java

```diff
@@ -0,0 +1,58 @@
+package com.cmput301w21t06.crowdfly.Controllers;
```

Comment:
This should probably have a citations since I originally got it from the discussion forum